frontend/src/v4/routes/components/dialogContainer/components/redirectToTeamspaceDialog/redirectToTeamspaceDialog.component.tsx

```diff
@@ -41,6 +42,6 @@ export const RedirectToTeamspaceDialog = ({ content, message, status, handleReso
 			<DialogActions>
 				<Button onClick={handleResolve} variant="contained" color="secondary">Back to teamspace</Button>
```

Comment:
I have made an issue for the "back to teamspace" button redirecting to the wrong url in v5 #3768